src/lib/createHelpers.js

```diff
@@ -1,7 +1,25 @@
+import highlight from './highlight';
+
 export default function({ numberLocale }) {
   return {
     formatNumber(number, render) {
       return Number(render(number)).toLocaleString(numberLocale);
     },
+    highlight(options, render) {
+      let highlightOptions;
+
+      try {
+        highlightOptions = JSON.parse(options);
+      } catch (error) {
+        return render('');
```

Comment:
We shouldn't fail silently.

Rather warn in the console or throw an error. WDYT?

Reply:
that should be an error, since if the JSON is wrong, that means the highlight will never work. 

If the highlighted attribute is not in attributesForHighlighting, that should be a warning (only in dev), like in Vue InstantSearch